src/lightning_app/core/flow.py

```diff
@@ -286,6 +287,15 @@ def flows(self) -> Dict[str, "LightningFlow"]:
             flows.update(getattr(self, struct_name).flows)
         return flows
 
+    @property
+    def lightningignore(self) -> List[str]:
```

Comment:
This doesn't really need a getter/setter. I added it just for the docstring. I can remove them if you prefer